src/_utils/generateDropdownOptions.js

```diff
@@ -5,3 +5,9 @@ export default arrayWithOptions => arrayWithOptions.map(option => ({
   text: option,
   value: option
 }));
+
+export const generateModalDropdownOptions = option => ({
```

Comment:
Just rename this as `generateDropdownOptions` as it can also be reused in a different component that is not a modal. Also, update the default export to:
```jsx
export default arrayWithOptions => arrayWithOptions.map(generateDropdownOptions);